.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+## Your Environment
+Thank you for taking the time to report an issue.
+
+To more efficiently resolve this issue, we'd like to know some basic information about your system and setup.
+
+1) Your OS (as much info as possible)
+* ie: Windows 10, macOS High Sierra version 10.13(17A365), Ubuntu 18.04 LTS, etc)
+
+2) Version of python you are running
+``` bash
+$ python --version
+```
+
+3) Version of twine that you are using (include complete output of):
+``` bash
+$ twine --version
```

Comment:
We'll also want to know how they installed `twine`. We should ask:

```
How did you install twine? Did you use your operating system's package manager or pip or something else?
```

Reply:
It seems you missed this comment. Please add these in and this should be good to add.

Reply:
Please prompt the user for the method with which they're installing twine. That's useful information.

Reply:
Sorry overlooked this earlier.

Thank you for being so patient w/ me all throughout this PR.